### ISSUE TITLE
can I get song data and play in my app with ShazamKit? 

### ISSUE THREAD

**Issue by xts-bit** [alanjhughes/expo-shazamkit#1]
can I get song data and play in my app with ShazamKit?

**Comment by alanjhughes** [comment]
You can play a snippet of the song but not the entire thing.

**Comment by xts-bit** [comment]
@alanjhughes I mean can i get details about the song by its name like searching etc?? without Listening just give it a name of song on shazam and it will give me that song details

**Comment by alanjhughes** [comment]
No, they don't expose apis to do text search for songs. We'd need to build a lib using `MusicKit` to do that

**Comment by xts-bit** [comment]
@alanjhughes Yep, But then how do people on RapidAPI get APIS for searching songs on Shazam and getting details about it? Any idea about this? or tool in mind?

**Comment by alanjhughes** [comment]
I'm not sure to be honest. Maybe Shazam had apis to do this but this library uses the apple provided `ShazamKit` so I can only use what it exposes. I need to do some work on this lib anyway so I'll see if anything has changed.
